docs/editors/overview.md

```diff
@@ -38,181 +39,207 @@ functionality.
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Rename symbol</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Code actions</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Worksheets</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Document symbols</td>
     <td align="center">✅</td>
     <td align="center">Flat</td>
     <td align="center">Flat</td>
     <td align="center">✅</td>
+    <td align="center">Flat</td>
   </tr>
   <tr>
     <td>Workspace symbols</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Formatting</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Folding</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center"> </td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Highlight</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Metals Extensions</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">Status bar, Input box, Decoration protocol, Did focus</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
      <td>Organize imports</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">❌</td>
   </tr>
   <tr>
      <td>Implicit decorations</td>
      <td align="center">✅</td>
      <td align="center">Shown in hover</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">❌</td>
   </tr>
   <tr>
      <td>Source file analyzer</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">❌</td>
   </tr>
   <tr>
      <td>Find text in dependency JAR files</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">❌</td>
+     <td align="center">✅</td>
   </tr>
   <tr>
      <td>Run scalafix rules</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">✅</td>
```

Comment:
Is it really possible to run scalafix rules from helix? One can see the `scalafix-run` and `scalafix-run-only` in Helix LSP workspace command picker, but how to actually run a scalafix command?

Reply:
Good question, I just saw them in there and assumed that it worked... have now tested and appears broken, getting `2024.02.14 22:17:48 ERROR Expected 'This command should be sent in with the LSP [`TextDocumentPositionParams`](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams)` will update the PR to reflect this